docs/extending/extending_udw.md

```diff
@@ -5,14 +5,28 @@ using an interactive interface: browse, search, create, and bookmarks view.
 
 ## How to use UDW?
 
+UDW requires configuration that you can provide using the `ez_udw_config` Twig helper. This configuration must be spread to the props of the component itself.
+
+```html+twig
+<button data-udw-config="{{ ez_udw_config('single') }}">
+    Open My UDW
+</button>
+```
+
+> `simple` configuration is one of the default configuration provided. You can also do your [own configuration](#Adding-new-configuration).
+
 With vanilla JS:
 
 ``` js
 const container = document.querySelector('#react-udw');
 
+const config = /* fetch the config somewhere */;
+//const config = JSON.parse(somewhere.dataset.udwConfig);
```

Comment:
I would write something like this:
```suggestion
//const config = JSON.parse(document.querySelector('.btn-udw-trigger).dataset.udwConfig);
```

And I would add this class to the button above to show which udwConfig should be passed to the React component